packages/core/test/endowmentsToolkit.spec.js

```diff
@@ -400,6 +470,30 @@ test('getEndowmentsForConfig - specify unwrap from, unwrap to', (t) => {
   t.is(getter.call(), globalThis)
 })
 
+test('endowAll - specify unwrap from, unwrap to', (t) => {
+  'use strict'
```

Comment:
Maybe move L493 up here so we can more easily understand that this is necessary and not just a way to make ESLint shut up.  Or is it?  From that comment, it sounds like it's needed at runtime, right? Would just like this clarified

[should]